redshift_connector/cursor.py

```diff
@@ -239,6 +239,51 @@ def executemany(self: "Cursor", operation, param_sets) -> "Cursor":
         self._row_count = -1 if -1 in rowcounts else sum(rowcounts)
         return self
 
+    def insert_data_bulk(self: "Cursor", filename,table_name, column_indexes,column_names,delimeter)-> "Cursor":
+  
+        """runs a single bulk insert statement into the database.
+           
+           This method is part of the `DBAPI 2.0 specification
+           <http://www.python.org/dev/peps/pep-0249/>`_.
```

Comment:
Is `insert_data_bulk` method defined in [DBAPI 2.0 reference](https://www.python.org/dev/peps/pep-0249/)? I'm not seeing it there. If it isn't, could we reword this to state that:

```
This method is native to redshift_connector.
```

Reply:
addressed in 8d114f1